vertx-core/src/main/java/io/vertx/core/http/HttpClientAgent.java

```diff
@@ -72,4 +81,13 @@ default Future<Boolean> updateSSLOptions(ClientSSLOptions options) {
    */
   Future<HttpClientConnection> connect(HttpConnectOptions options);
 
+  /**
+   * Provides a way for users of the client to get access to the default redirect handler.
+   * This is useful for building redirect handlers that need to delegate to the default behavior.
+   */
```

Comment:
I prefer not having this method, users should care about the handler when using the builder

Reply:
Removed